presentation/src/main/java/com/dcs/presentation/ui/mediadetail/component/Casts.kt

```diff
@@ -71,6 +72,9 @@ internal fun Casts(
                             .width(140.dp)
                             .aspectRatio(0.6f)
                             .padding(horizontal = 6.dp)
+                            .clickable {
+                                onPersonClick(casts[index].id)
+                            }
```

Comment:
람다로 새로 열지 말고 아래와 같이 하면 리컴포지션 할 때 스마트 리컴포지션으로 스킵된다고 해요. 람다로 열었을 때 람다 내부에 unstable한 변수 참조하면 리컴포지션 될 떄마다 리컴포지션 된다고 합니다.
```suggestion
                            .clickable(onClick = onPersonClick(casts[index].id))
```

Reply:
해당 형식은 () -> Unit 이것만 되는 것 아닌가??

Reply:
{} <- 요건도 해당됩니다. 그래서 reference(::) 를 하던가 위와 같이 최대한 작성하라고 하더라구요

Reply:
Modifier의 clickable에 onClick: () -> Unit 이 이렇게 선언되어 있는데 어떻게 한다는거지??

Reply:
아 그 얘기셨구나ㅋㅋ안되는군요